### DISCUSSION TITLE
Make multiplayer scoreboard separately hideable

### DISCUSSION THREAD

**Discussion by farkasma** [ppy/osu#17747] [Ideas] [answered]
Currently the multiplayer scoreboard is only hideable together with the rest of the HUD.
I feel like the scoreboard, possibly along with the chat box, should be hideable.
The current design of the scoreboard hangs over my playfield, obscuring the top left portion of it. The small icons that appear when HUD is hidden could be an option, but the best would imo be the ability to hide just that on it's own, just like in stable.

**Comment by peppy** [comment, marked as answer]
This will be changing in the near future, with an option to select which elements are visible only during break time.

**Reply by janneslessmann** [reply]
That would be really cool!

**Reply by peppy** [reply]
Should be resolved for now, via https://github.com/ppy/osu/pull/18221.

**Reply by janneslessmann** [reply]
Will there be an option in the future, where you will be able to hide it completely (not just putting it into "compact mode")?

**Reply by peppy** [reply]
Likely no.

**Reply by janneslessmann** [reply]
It would be really helpful. Many users (me & some of my friends included) get distracted very easily and some users may just like a complete clean screen without any HUD elements on it.

**Reply by smoogipoo** [reply]
I think last time the discussion hinted at this being a multiplayer room setting in the future.
